### PR TITLE
Added pointer pages, to have a 2nd location of "webhint" & "DevTools Protocol" in TOC

### PR DESCRIPTION
**Pull request by mikehoffms**
**Rendered pointer pages for review:**
https://review.docs.microsoft.com/en-us/microsoft-edge/test-and-automation/devtools-protocol?branch=pr-en-us-1471
https://review.docs.microsoft.com/en-us/microsoft-edge/test-and-automation/webhint?branch=pr-en-us-1471

With this PR, the "Test and automation" bucket of TOC now has "DevTools Protocol" as a short pointer page, and "webhint" as a short pointer page.

### CODE REVIEW

File: microsoft-edge/toc.yml [hunk 1]
```diff
@@ -530,7 +530,7 @@
         - name: Accessibility testing
           href: accessibility/test.md
 # SORT into the above card sections ===========================================
-    - name: Privacy whitepaper
```
Comment (Reezaali) [inline microsoft-edge/toc.yml:533]: Can we revert please, unless CELA & Privacy requested this change?

File: microsoft-edge/test-and-automation/devtools-protocol.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+description: Overview of Microsoft Edge DevTools Protocol.
+title: DevTools Protocol
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 11/03/2021
+ms.topic: reference
```
Comment (Reezaali) [inline microsoft-edge/test-and-automation/devtools-protocol.md:7]: Can we confirm that reference is API ref only? I think ms.topic should be conceptual.

File: microsoft-edge/test-and-automation/devtools-protocol.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+description: Overview of Microsoft Edge DevTools Protocol.
+title: DevTools Protocol
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 11/03/2021
```
Comment (Reezaali) [inline microsoft-edge/test-and-automation/devtools-protocol.md:6]: Can we update the dates?

File: microsoft-edge/test-and-automation/webhint.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+---
+description: About the webhint extension for Visual Studio Code.
+title: webhint extension for Visual Studio Code
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 11/03/2021
+ms.topic: article
```
Comment (Reezaali) [inline microsoft-edge/test-and-automation/webhint.md:7]: can we update the date? 
Does article need to change to conceptual?

File: microsoft-edge/test-and-automation/webhint.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+ms.prod: microsoft-edge
+keywords: microsoft edge, web development, vs code, visual studio code, webhint
+---
+# webhint extension for Visual Studio Code
+
+Use [webhint](https://webhint.io), a customizable linting tool, to improve the accessibility, performance, cross-browser compatibility, PWA compatibility, and security of your site.  The webhint extension checks your code for best practices and common errors. This open-source project, initially developed by the Microsoft Edge team, is now part of the [OpenJS Foundation](https://openjsf.org).  The Microsoft Edge team continues to contribute to webhint alongside web developers in the community.
+
+Identify and fix problems in your HTML, CSS, JavaScript, TypeScript, and more by adding the webhint extension for Visual Studio Code.  Hints appear as inline underlines and are summarized in the **Problems** pane.
```
Comment (Reezaali) [inline microsoft-edge/test-and-automation/webhint.md:15]:
```suggestion
Identify and fix problems in your HTML, CSS, JavaScript, TypeScript, and more by installing the webhint extension for Visual Studio Code.  Hints appear as inline underlines and are summarized in the **Problems** pane.
```

File: microsoft-edge/test-and-automation/webhint.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+
+:::image type="content" source="../visual-studio-code/media/webhint-extension.png" alt-text="The webhint extension for Visual Studio Code.":::
```
Comment (Reezaali) [inline microsoft-edge/test-and-automation/webhint.md:17]: can we add redlines around what is helpful to focus on? Otherwise should we remove it?

**Review by Reezaali — COMMENTED** [review]
Left a couple of comments for your review.

File: microsoft-edge/test-and-automation/devtools-protocol.md (hunk 2 above)
Comment (mikehoffms) [reply microsoft-edge/test-and-automation/devtools-protocol.md:7]: Changed to "conceptual".  For the value "reference", internal DCG reads "Any authored (read: not autogenerated) content placed under a Reference node in a ToC."  This article was in a small, 4-child Reference node until recently.  The page contains reference-type information about HTTP endpoints (URL path patterns).  The connotations of the word 'reference' are shifting or broadening from API Reference to include REST Endpoint Reference.  DCG only defines the values in terms of the TOC bucket label, "Reference", and mentions "autogenerated" vs. (manually) "authored" pages.

File: microsoft-edge/test-and-automation/webhint.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 11/09/2021
+ms.topic: conceptual
+ms.prod: microsoft-edge
+keywords: microsoft edge, web development, vs code, visual studio code, webhint
+---
+# webhint extension for Visual Studio Code
+
+Use [webhint](https://webhint.io), a customizable linting tool, to improve the accessibility, performance, cross-browser compatibility, PWA compatibility, and security of your site.  The webhint extension checks your code for best practices and common errors. This open-source project, initially developed by the Microsoft Edge team, is now part of the [OpenJS Foundation](https://openjsf.org).  The Microsoft Edge team continues to contribute to webhint alongside web developers in the community.
```
Comment (Reezaali) [inline microsoft-edge/test-and-automation/webhint.md:13]:
```suggestion
Use [webhint](https://webhint.io), a customizable linting tool, to improve the accessibility, performance, cross-browser compatibility, PWA compatibility, and security of your site.  The webhint extension checks your code for best practices and common errors. 
```

File: microsoft-edge/test-and-automation/webhint.md [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+
+Identify and fix problems in your HTML, CSS, JavaScript, TypeScript, and more by installing the webhint extension for Visual Studio Code.  Hints appear as inline underlines and are summarized in the **Problems** pane.
```
Comment (Reezaali) [inline microsoft-edge/test-and-automation/webhint.md:15]:
```suggestion

```

File: microsoft-edge/test-and-automation/webhint.md [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+
+:::image type="content" source="../visual-studio-code/media/webhint-extension.png" alt-text="The webhint extension for Visual Studio Code.":::
+
```
Comment (Reezaali) [inline microsoft-edge/test-and-automation/webhint.md:18]:
```suggestion

```

**Review by Reezaali — APPROVED** [review]
Left some minor recommendations.

**Pull request merged**